types/go.mod

```diff
@@ -2,9 +2,14 @@ module github.com/sensu/sensu-go/types
 
 go 1.13
 
+replace (
+	github.com/sensu/sensu-go => ../sensu-go
+	github.com/sensu/sensu-go/api/core/v2 => ../api/core/v2
+)
```

Comment:
Is this intended?

Reply:
Yes, the types package should import whatever the "current" version of sensu-go and core/v2 are.

Reply:
Ah I see, thanks for the explanation!